templates/candidates-single.html

```diff
@@ -58,8 +58,8 @@ <h4 class=>Party</h4>
             {% if leadership_committees %}
             <dl>
               <dt>Leadership PACs</dt>
-              {% for committee in leadership_committees %}
-              <dd><a {% if committee.name_url %}href="{{ committee.name_url }}"{% endif %}>{{ committee.name }}</a></dd>
+              {% for c_id in leadership_committees %}
+              <dd><a {% if leadership_committees[c_id].url %}href="{{ leadership_committees[c_id].url }}"{% endif %}>{{ leadership_committees[c_id].name }}</a></dd>
```

Comment:
Should the link be here at all? This'll generate an <a> with no href if no url is present, right? Perhaps that's a super rare case